### PR TITLE
OnnxLoader supports Gemm

### PR DESCRIPTION
**Pull request by idow09**

### CODE REVIEW

**Comment by idow09** [comment]
Hi! I'm not sure if my choices are the best as of what involves the python side. Currently I used the existing failing "end2end" test to check simpler models but I override the mnist.onnx so maybe I should separate it with relevant naming etc.
Please let me know what u think!

File: src/ConvNetSharp.Flow/Serialization/Onnx/OnnxLoader.cs [hunk 1]
```diff
@@ -156,6 +161,31 @@ private Op<T> GetConvOp(NodeProto node, Dictionary<string, Op<T>> seen)
...
+            var weightVolume = BuildVolume(this._initializers[node.Inputs[1]]);
+            var biasVolume = BuildVolume(this._initializers[node.Inputs[2]]);
+            var weight = this._cns.Variable(weightVolume, $"Filter_{weightVolume.Shape.Dimensions[3]}", true);
+            var bias = this._cns.Variable(biasVolume, "Bias", true);
+
+            var data = new Dictionary<string, object>
+            {
+                ["Alpha"] = node.Attributes.First(o => o.Name == "alpha").F, // TODO: use alpha
+                ["Beta"] = node.Attributes.First(o => o.Name == "beta").F, // TODO: use beta
+                // ["TransA"] = node.Attributes.First(o => o.Name == "transA").I.ToString(), // TODO: use transA
```
Comment (cbovar) [inline src/ConvNetSharp.Flow/Serialization/Onnx/OnnxLoader.cs:177]: Why is it commented ?

**Comment by cbovar** [comment]
Hi! This great :) I had totally forgotten `MatMult` op was implemented. 
I guess eventually we should have an onnyx export for each basic operation with its corresponding C# unit test.

I understand to have a full Gemm support, we need to take into account `alpha`, `beta`, `transA`, `transB`. (those parameters are handled by this cudnn [methods](https://github.com/cbovar/ConvNetSharp/blob/master/src/ConvNetSharp.Volume.GPU/Double/Volume.cs#L574)) but I want to go ahead and merge this into Onnx branch if that's ok with you.

**Comment by idow09** [comment]
Of course it's okay. I was afraid it ain't good enough because it's not thoroughly tested. But I agree we should focus on completing the conversions so we have a full op set and in the meanwhile improve the testing mechanism.
Go ahead and merge :)

File: src/ConvNetSharp.Flow/Serialization/Onnx/OnnxLoader.cs (hunk 1 above)
Comment (idow09) [reply src/ConvNetSharp.Flow/Serialization/Onnx/OnnxLoader.cs:177]: Oh! This is important. These parameters aren't always present (in my case only `transA` was missing) so the code should handle checking for their existence. I'm new to C# and I wasn't sure what is the best approach for that...

**Pull request merged**